### PR TITLE
feat: Added Contributors banner

### PR DESCRIPTION
**Pull request by indujaaaa**
## Fixes Issue

This PR fixes the following issues : #35 

## Changes proposed
Added a banner of Contributors at the footer of the README.

## Check List (Check all the boxes which are applicable)

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarised content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots 
Contributors Banner

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,13 @@ Whenever you want to contribute to any project it is a good practice to make a s
...
-### Maintainer :
+## Contributors :
+
+Every contributor's efforts and time are deeply appreciated :smile:
 
+<a href = "https://github.com/kaiwalyakoparkar/classroom-monitor-bot/graphs/contributors">
+  <img src = "https://contrib.rocks/image?repo=kaiwalyakoparkar/classroom-monitor-bot"/>
+</a>
+
+### Maintainer :
```
Comment (kaiwalyakoparkar) [inline README.md:133]: Remove this maintainer section as you added the contributor's list already :smile:

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (indujaaaa) [reply README.md]: Done! @kaiwalyakoparkar

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for collaborating and contributing.

**Comment by kaiwalyakoparkar** [comment]
@indujaaaa would you please resolve this conflict?

**Comment by kaiwalyakoparkar** [comment]
@indujaaaa So sorry for that. I by mistakenly closed it :sweat_smile: You can resolve the conflict now

**Comment by indujaaaa** [comment]
@kaiwalyakoparkar I'm new to contributing, but I think I've resolved it. Please check and let me know if there's anything I should do. Sorry if I'm dragging a simple thing for long 😅

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for your collaboration and contribution.

**Pull request merged**